test/spec/unit/utils/ipUtils_spec.js

```diff
@@ -0,0 +1,58 @@
+import { scrubIPv4, scrubIPv6 } from '../../../../src/utils/ipUtils'
+
+describe('ipUtils', () => {
+  describe('ipv4', () => {
+    it('should mask ip v4', () => {
+      let input = '192.168.1.1';
+      let output = scrubIPv4(input);
+      expect(output).to.deep.equal('192.168.1.0');
+      input = '192.168.255.255';
+      output = scrubIPv4(input);
+      expect(output).to.deep.equal('192.168.255.0');
+    });
+
+    it('should convert null to blank string', () => {
+      let input = null;
+      let output = scrubIPv4(input);
+      expect(output).to.deep.equal('');
+    });
```

Comment:
not sure if this was prompted by my comment - I meant almost the opposite, a null ip should not be transformed into an empty string (since an empty string is not a valid ip).

It doesn't matter for this because the redaction rule [only runs when the field is non-null](https://github.com/prebid/Prebid.js/blob/e0651f657bd6d1a3dfa8053df007d97cf8034b1d/src/activities/redactor.js#L53).  However if we end up re-using the scrubbing utilities in the future, I think this will prove to be an odd choice. I'll leave it to you whether you want to change it. (IMO returning null or throwing would both be better).